chore: post rewrite fixes

savioxavier opened the pull request:
This PR updates/changes the following stuff:
- Updated styles and wording for multiple components
- Changed monospace font to Fira Code
- Fixed code mockup issues
- Fixed some responsiveness issues
- Added two new fonts: Open Sans and Cabin to override system default
	- These two were the best pairings for Montserrat I could find
- Fixed some unnecessary bolding of text
- Formatted and cleaned up some files

savioxavier committed:
chore: update styles

Updated styles and wording for multiple components

savioxavier committed:
style: format files using prettier

savioxavier committed:
feat: use Fira Code as monospace font

savioxavier committed:
fix: text center issues on smaller viewports

savioxavier committed:
feat: add new fonts to override system defaults

savioxavier committed:
fix: text center issues and code cleanup

savioxavier committed:
chore: code cleanup

savioxavier commented:
@megatank58 Can I use FontAwesome to render icons? I know it's a huge library, but it was kind of tough to work with SVGs, and FontAwesome's font implementation is so much easier. Plus there is more than one icon in the site, and more icons might be added too for aesthetic purposes.

megatank58 commented:
Does font awesome only bundle the icons we use? if it doesn't bundles all icons then I'm fine with using it

savioxavier committed:
chore: add font awesome icons

savioxavier commented:
I was unable to resolve solid Font Awesome icons for some reason, got the Golang icon ready.

savioxavier committed:
Merge branch 'main' into post-rewrite-fixes

megatank58 commented:
Linter workflow is failing, lint the codebase

savioxavier committed:
build: add script lint:check

megatank58 commented on src/App.vue:
```diff
@@ -1,15 +1,14 @@
+	<Navbar />
+	<router-view />
+	<Footer />
+	<MadeBy />
```
MadeBy component isn't imported here

megatank58 commented on src/components/Install.vue:
```diff
@@ -40,16 +41,15 @@ export default defineComponent({
 			this.setupRelatedCode = parseMarkdownColors(
-				`
-	package main
+				`	package main
 
 	import "github.com/gominima/minima"
 	
 	func main() {
 		app := minima.New()
 		app.Listen(":3000")
 	}
-			`.trim(),
+			`,
```
why does `package main` has so many tabs and why was trim removed

savioxavier commented:
> Linter workflow is failing, lint the codebase

Fixed!

megatank58 commented:
just some things, rest looks good to me

apoorvcodes commented:
@savioxavier I don't like the fonts, too thin.

savioxavier commented:
> @savioxavier I don't like the fonts, too thin.

I just selected a few fonts in order to override the system defaults (it doesn't look too good on Android). I couldn't find any other better font combos for Montserrat, so I used Cabin and Open Sans. The font weight is the _default_ font weight, provided by all the fonts, but that could be adjusted later. If you wish, you can suggest or provide other fonts.

The initial font weights which you used for Montserrat (weight 700) struck me as _too bold_, so I resorted to the default weight, meaning to adjust it later.

savioxavier committed:
chore: update fonts and font weights

savioxavier committed:
feat: use Pacifico font to render logo

savioxavier commented:
@apoorvcodes Deleted the Cabin font, updated font widths to make them bold, and added Pacifico font to render the logo.

You can always add new fonts/change them after this PR is merged
